apps/Conduit-UI/src/features/database/components/schemas/Schemas.tsx

```diff
@@ -296,18 +296,18 @@ const Schemas: FC = () => {
                     title={
                       <Box display="flex" flexDirection="column" gap={2} p={2}>
                         <Typography variant="body2">
-                          Custom content types are expressed as database schemas. Schemas contain
-                          fields of different types. Some of them are optional, while others are
-                          required. Some of them might even take a default value if nothing else is
-                          explicitly specified.
+                          Custom content types are expressed as Models which are created in the
```

Comment:
I'd rephrase this a bit, but more specifically is the capital M intended?

ex:
Custom content types are expressed through models, represented as database schemas. Models contain fields of different types, some of which are mandatory while others are optional. Some of them may even accept a default value if none is explicitly provided.